gridstatus/ercot.py

```diff
@@ -3430,6 +3430,21 @@ def read_docs(
             )
         return pd.concat(dfs).reset_index(drop=True)
 
+    def ambiguous_based_on_dstflag(self, df: pd.DataFrame) -> pd.Series:
```

Comment:
Moved this to a top-level function in this class so it can be used by `ErcotAPI`.